tests/test_study.py

```diff
@@ -554,6 +554,19 @@ def objective(trial):
     study.optimize(objective, n_trials=10, catch=())
 
 
+@pytest.mark.parametrize('storage_mode', STORAGE_MODES)
+def test__append_trial(storage_mode):
```

Comment:
`_` is duplicated.

Reply:
Thank you for pointing it out.
Removed: https://github.com/pfnet/optuna/pull/464/commits/90103b1a86023dc45c5afb9a4a62b1b8cb87bb3f